torch_dicom/preprocessing/pipeline.py

```diff
@@ -19,6 +19,10 @@
 
 from .. import __version__
 from ..datasets import AggregateInput, collate_fn, uncollate
+from ..datasets.image import save_image
+
+
+OUTPUT_FORMATS: Final = ("png", "dcm")
```

Comment:
Perhaps this should be an enum? Then we can replace instances of "dcm" with `OUTPUT_FORMATS.DICOM` (or similar).